pendulum:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 1000  


  # Effort Controllers ---------------------------------------
  
  joint1_torque_controller:
    type: effort_controllers/JointEffortController
    joint: joint1
  joint2_torque_controller:
    type: effort_controllers/JointEffortController
    joint: joint2
  #joint3_torque_controller:
  # type: effort_controllers/JointEffortController
  # joint: joint3
  # pid: {p: 100.0, i: 0.01, d: 10.0}

